paths/gridPath/README.md

```
@@ -0,0 +1,74 @@
+      /* options */
+    },
+  });
+})();
+```
+
+### ESM / CommonJS
+
+This package is compatible also with ES or CommonJS modules, firstly this needs to be installed, like this:
+
+```shell
+$ npm install @tsparticles/path-grid-path
```

markdownlint-cli2 (0.20.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)